components/alert/alert.css

```diff
@@ -0,0 +1,78 @@
+:root{
+    --alert-error: #f5c0c6;
+    --alert-success: #d0f0e1;
+    --alert-warning: #f4dfc8;
+    --alert-info: #bde3f7;
+    --alert-error-outline: #f25b6d;
+    --alert-success-outline: #468367;
+    --alert-warning-outline: #f4a653;
+    --alert-info-outline: #4f91b5;
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
```

Comment:
Move this code to a separate CSS file and then import that file everywhere using @import, as these are common values that you may require later for each component

Reply:
Hey, I have made all the required changes.